tika-parsers/src/test/resources/test-properties/TesseractOCRConfig-full.properties

```diff
@@ -20,3 +20,10 @@ pageSegMode=2
 maxFileSizeToOcr=2000000
 timeout=240
 minFileSizeToOcr=1
+
+enableProcessing=1
+ImageMagickPath=/usr/local/bin
```

Comment:
I think that this should be set to null value. This will not work on Window for example.